packages/common/object/src/map-object.ts

```diff
@@ -0,0 +1,16 @@
+import { PlainObjectOf } from '@angular-ru/common/typings';
+
+export function mapObject<Type, ReturnType>(
+    object: PlainObjectOf<Type>,
```

Comment:
`PlainObjectOf<Type>` not `Type`?

Reply:
shto

Reply:
why `export function mapObject<T, K>(obj: PlainObject<T>)` not `export function mapObject<T, K>(obj: T)`